client/lib/importer/importer-config.js

```diff
@@ -23,16 +23,12 @@ function getConfig( { siteTitle = '' } = {} ) {
 		icon: 'wordpress',
 		description: translate( 'Import posts, pages, and media from a WordPress export\u00A0file.' ),
 		uploadDescription: translate(
-			'Upload a {{b}}WordPress export file{{/b}} to start ' +
-				'importing into {{b2}}%(title)s{{/b2}}. A WordPress export is ' +
+			'A WordPress export is ' +
```

Comment:
I think it's a bit of a stress case, since (as I understand it) most folks have one website with us, but I see what you mean. This field doesn't feel like the right place to emphasize the site title; it feels more instructional, explaining what a WordPress export file is.

What if we added the site title to the `description` instead?  Ie "Import posts, pages, and media from a WordPress export file to [[site title]]" -- that way it stays on the page as you're going through the process.

Reply:
That seems like a good solution. Thanks!